ui/pages/settings/snaps/view-snap/view-snap.js

```diff
@@ -18,26 +18,42 @@ import SnapAuthorshipExpanded from '../../../../components/app/snaps/snap-author
 import Box from '../../../../components/ui/box';
 import SnapRemoveWarning from '../../../../components/app/snaps/snap-remove-warning';
 import ConnectedSitesList from '../../../../components/app/connected-sites-list';
-
+///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
+import KeyringSnapRemovalWarning from '../../../../components/app/snaps/keyring-snap-removal-warning';
+///: END:ONLY_INCLUDE_IN
 import { SNAPS_LIST_ROUTE } from '../../../../helpers/constants/routes';
 import {
   removeSnap,
   removePermissionsFor,
   updateCaveat,
+  ///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
+  showKeyringSnapRemovalModal,
+  ///: END:ONLY_INCLUDE_IN
 } from '../../../../store/actions';
 import {
   getSnaps,
   getSubjectsWithSnapPermission,
   getPermissions,
   getPermissionSubjects,
   getTargetSubjectMetadata,
+  ///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
+  getKeyringSnapAccounts,
+  ///: END:ONLY_INCLUDE_IN
 } from '../../../../selectors';
 import { getSnapName } from '../../../../helpers/utils/util';
 import { Text } from '../../../../components/component-library';
 import SnapPermissionsList from '../../../../components/app/snaps/snap-permissions-list';
 import { SnapDelineator } from '../../../../components/app/snaps/snap-delineator';
 import { DelineatorType } from '../../../../helpers/constants/snaps';
 
+///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
+const KeyringSnapRemovalResultStatus = {
+  Success: 'success',
+  Failed: 'failed',
+  None: 'none',
+};
+///: END:ONLY_INCLUDE_IN
+
```

Comment:
Can we move this enum to a different file?

Reply:
Sure, i've made the change in 899aba2246a6d420857c7a4de58584e5569a7604